webpack.config.js

```diff
@@ -17,8 +17,11 @@ module.exports = {
     libraryTarget: 'commonjs2',
     path: path.join(__dirname, '.webpack'),
     filename: '[name].js',
+    sourceMapFilename: '[file].map',
   },
-  externals: [nodeExternals()],
```

Comment:
GitHub actions: lint-format-test

'nodeExternals' is assigned a value but never used

Reply:
thx, will fix it